components/konflux-ui/staging/base/proxy/proxy.yaml

```diff
@@ -272,6 +272,14 @@ rules:
 - apiGroups: ["authorization.k8s.io"]
   resources: ["localsubjectaccessreviews"]
   verbs: ["create"]
+- apiGroups:  # TEMPORARY SOLUTION TO UNBLOCK KONFLUX-166 WHILE KAR-510 IS NOT SOLVED
```

Comment:
how this change is going to solve the issue?
are you going to use the proxie's identity when querying for snapshots and releases? if so, I don't think we should allow it.

Reply:
> are you going to use the proxie's identity when querying for snapshots and releases? if so, I don't think we should allow it.

Yes, that's what I'm trying to do here and I'm aware that this is not how this should work.
I'm asking to do this *temporarilly* while kubearchive doesn't support impersonation just for snapshots and releases and just in staging so the UI team is unblocked for their development.

That's why I added the comment, so it's clear that this piece shouldn't be there, this should be considered a temporary exception.

If I'm not mistaken, this will impact only for the access to the archived snapshots and releases, not the live ones so I think that the impact is quite delimited.

Reply:
no, it will give all the users access to all the snapshots and releases on the cluster.

Reply:
ok I see what you mean. Why not investing few more hours an configure https://github.com/openshift/kube-rbac-proxy?tab=readme-ov-file  that will do the token/access review for kubearchive.